lib/jerboa/client.ex

```diff
@@ -121,6 +121,22 @@ defmodule Jerboa.Client do
   end
 
   @doc """
+  Creates permission on the allocation for the given peer
+  address
+
+  If permission is already installed for the given address,
+  the permission will be refreshed.
+
+  ## Example
+
+      iex> create_permission client, {192, 168, 22, 111}
+  """
+  @spec create_permission(t, peer :: ip) :: :ok | {:error, error}
```

Comment:
We can't handle multiple permissions yet because of `Params` module. `put_atrr` will override attribute if such attribute is already present in struct, and `get_attr` will return only first occurence of attribute.